fusor-ember-cli/app/templates/components/naming-scheme-modal.hbs

```diff
@@ -14,7 +14,11 @@
                                   value=hostNamingScheme
                                   disabled=isStarted
                                   labelSize='col-md-4'
-                                  inputSize='col-md-5'}}
+                                  inputSize='col-md-5'
+                                  disabled=isStarted
+                                  renderInPlace=true
```

Comment:
This is need for dropdowns in modals. 
http://www.ember-power-select.com/docs/troubleshooting